noxfile.py

```diff
@@ -155,6 +156,14 @@ def pytype(session):
     session.run("pytype")
 
 
+@nox.session(python="3.6")
```

Comment:
Why not DEFAULT_PYTHON_VERSION?

Reply:
I'm matching what is in the `mypy.ini` file I lifted from @crwilcox.

Reply:
that specifies to mypy what minimum version of python the types should be compatible with. If I understand correctly we should be able to run against a later python version